tests/nfv/run_integration_tests.pm

```diff
@@ -16,18 +16,13 @@ use base "consoletest";
 use testapi;
 use strict;
 use utils;
-use lockapi;
 use mmapi;
 
 sub run {
     my $self        = shift;
     my $vsperf_conf = "/etc/vsperf_ovs.conf";
 
     select_console 'root-console';
-    mutex_create('nfv_trafficgen_ready');
```

Comment:
just to keep code consistent , if you deleting mutex in one place do the same on another. Or introduce new place where you will use it. Currently your code regarding nfv_trafficgen_ready in moongen_installation doesn't make sense

Reply:
Right, I forgot to remove it from moongen. The mutex will be needed before running the tests.   Let me correct it.